scripts/lochness_create_template.py

```diff
@@ -295,7 +295,7 @@ def create_config_template(config_loc: Path, args: object) -> None:
 ssh_user: {args.ssh_user}
 ssh_host: {args.ssh_host}
 sender: {args.email}
-mindlamp_days_to_pull: 10
+mindlamp_days_to_pull: 100
```

Comment:
Please double check if 100 is intentional.

Reply:
Thanks for checking this. Since `mindlamp` server is not ready for the Pronet and Prescient, let's review this again in the future PR when we test Pronet `mindlamp` server.